Styles/Classes/Extensions/UIImage+Attachment.swift

```diff
@@ -5,7 +5,12 @@ import UIKit
 extension UIImage {
     func attachmentString(with style: TextStyle, baseAttributes: [NSAttributedStringKey: Any]) -> NSAttributedString {
         let attachment = NSTextAttachment()
-        attachment.image = self
+        let attributes = style.attributes
+        if let tintColor = attributes[.foregroundColor] as? UIColor, renderingMode != .alwaysOriginal {
```

Comment:
shouldn't it be invalid parameter if the rendering mode is set to always original ? Probably we should change it in this scope

Reply:
The idea was to tint image only if that is intented to do so. You can use some text style that has .foregroundColor which you don't want and bunch of other attributes that you are interested in but you don't want image to be tinted. In this case we choose to tint based on image rendering mode. 
So a default rendering mode for image attachment is to tint which means that we tint any image that has rendering mode `default` or `alwaysTemplate`.